AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLModelBasedTests.swift

```diff
@@ -294,6 +295,54 @@ class GraphQLModelBasedTests: XCTestCase {
         wait(for: [completeInvoked], timeout: TestCommonConstants.networkTimeout)
     }
 
+    func testUpdatePostWithInvalidCondition() {
+        let uuid = UUID().uuidString
+        let testMethodName = String("\(#function)".dropLast(2))
+        let title = testMethodName + "Title"
+        guard let createPost = createPost(id: uuid, title: title) else {
+            XCTFail("Failed to ensure at least one Post to be retrieved on the listQuery")
+            return
+        }
+        let conditionTitle = "IdontCareUpdated"
+        let failureInvoked = expectation(description: "request failed")
+
+        let post = Post.keys
+        let condition = post.title == conditionTitle
+
+        _ = Amplify.API.mutate(of: createPost, where: condition, type: .update, listener: { event in
+            switch event {
+            case .completed(let data):
+                switch data {
+                case .success:
+                    XCTFail("This test should fail since we are expecting the condition check to fail on backend")
```

Comment:
i think we can reduce this message without losing any the information, making it more concise:
"Received successful response data but failure with conditional check error"
- No need to say "this test should fail" since it is in the context of running the test
- "condtion[al] check" 
- "on backend" also seems like extra info since this is only an integration with the one service

Reply:
Changed to "XCTFail("Received successful response data but expecting conditional check failed")"